proposals/description/fip_99.ts

```diff
@@ -21,22 +21,30 @@ const fip_99: ProposalDescription = {
       target: 'core',
       values: '0',
       method: 'grantMinter(address)',
-      arguments: ['{globalRateLimitedMinter}'],
-      description: 'Grant the MINTER_ROLE to the global rate limited minter'
+      arguments: ['{raiPriceBoundPSM}'],
+      description: 'Grant the MINTER_ROLE to the raiPriceBoundPSM'
```

Comment:
it doesn't actually need the minter role because its redeem only.

Reply:
Yes but we might want to use it the other way in the future, right?

Reply:
probably not